web/src/modules/http/request.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { Observable, throwError } from "rxjs";
+import { catchError, map } from "rxjs/operators";
+import * as _ from "lodash";
```

Comment:
Please import only relevant packages from lodash.

Reply:
for this i will check and update my pr tomorrow